rewrite-core/src/test/java/org/openrewrite/RecipeEstimatedEffortTest.java

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite;
+
+import com.opencsv.CSVReaderHeaderAware;
+import com.opencsv.exceptions.CsvValidationException;
```

Comment:
Instead of this library, were you aware of the RecipeSpec methods to validate data table rows produced?
https://github.com/openrewrite/rewrite/blob/5f2d7e4c39876a987190278690ce3342ce0e0065/rewrite-test/src/main/java/org/openrewrite/test/RecipeSpec.java#L209-L252

Reply:
Up to you whether you'd want to see this merged or not; I think you've effectively shown the problem lies elsewhere, but we could merge these still to codify the expectations.